### PR TITLE
LFR-1924: Running flow when committing and in CI

### PR DESCRIPTION
**Pull request by lukeamarsh**
- Added a linting stage 
- Running linting stage in travis and Now
- Added pre-commit hook

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,20 +24,29 @@
     "test": "jest --projects packages/*",
     "bump": "lerna publish --exact --skip-npm --since \"v$(npm info @appearhere/bloom version)\"",
     "test:travis": "yarn test --coverage --maxWorkers=4",
-    "now-build": "yarn build && yarn build-storybook"
+    "now-build": "yarn build && yarn lint && yarn build-storybook",
+    "lint": "cd packages/core && flow"
```
Comment (zlahham) [inline package.json]: Can you name this as `lint-flow` or `flow`?

Comment (lukeamarsh) [reply package.json]: ah yes good shout @zlahham. Updated!

**Comment by lukeamarsh** [comment]
> We can handle flow type checking via github actions rather than Travis

How can we do that?

File: package.json [hunk 2]
```diff
@@ -22,20 +24,29 @@
     "test": "jest --projects packages/*",
     "bump": "lerna publish --exact --skip-npm --since \"v$(npm info @appearhere/bloom version)\"",
     "test:travis": "yarn test --coverage --maxWorkers=4",
-    "now-build": "yarn build && yarn build-storybook"
+    "now-build": "yarn build && yarn lint:flow && yarn build-storybook",
```
Comment (desmondhume) [inline package.json]: why do we need to lint it on now too?

Comment (lukeamarsh) [reply package.json]: I thought it might be nice to have it there too so they both fail for the same reason

Comment (desmondhume) [reply package.json]: I think it'd be useful to have it deployed regardless of flow, since we have at least one failing check

**Pull request merged**